src/classes/CRLP_Rollup_TDTM.cls

```diff
@@ -161,9 +161,23 @@ public class CRLP_Rollup_TDTM extends TDTM_Runnable {
             }
             opps = [SELECT Id, AccountId, Primary_Contact__c, npe03__Recurring_Donation__c FROM Opportunity WHERE Id IN :oppIds];
 
+        } else if (detailObjType == Account_Soft_Credit__c.SObjectType) {
```

Comment:
As I noted in the Queueable class, we should not attempt to rollup soft credits via a trigger because it will not work with a simultaneous hard credit rollup. The only alternative would be to find a way to chain the rollups, but NPSP has always stated that Soft Credit rollups are only calculated nightly. I don't think it's worth trying to crack that nut in this story. It's a much larger project of its own, if it's even possible.